adhocracy-plus/templates/500.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n static %}
 <html>
 <head>
-    <title>Error &mdash; {% trans 'adhocracy+' %}</title>
+    <title>Error &mdash; {{ settings.a4_candy_cms_settings.OrganisationSettings.platform_name }}</title>
```

Comment:
I am not sure we should use it in the 500 page? It might be broken. I'll try to check.

Reply:
Alright, I didn't find out. Even on master, when I broke the base template, there was no styling. 
(I did:
set DEBUG=False and add ALLOWED_HOSTS = ['localhost'] in/to settings/dev.py
start server with: python manage.py runserver 8004 --insecure 
also started the npm and changed a template tag in the base.html)
So, I will merge now and think about it again another time.